packages/terra-app-delegate/tests/jest/AppDelegate.test.jsx

```diff
@@ -232,13 +262,57 @@ describe('isEqual', () => {
     const goBack = () => {};
     const maximize = () => {};
     const minimize = () => {};
+    const requestFocus = () => {};
+    const releaseFocus = () => {};
 
     const appDelegateInstance1 = AppDelegate.create({
-      disclose, dismiss, closeDisclosure, goBack, maximize, minimize,
+      disclose, dismiss, closeDisclosure, goBack, maximize, minimize, requestFocus, releaseFocus,
     });
 
     const appDelegateInstance2 = AppDelegate.create({
-      disclose, dismiss, closeDisclosure, goBack, maximize, minimize: () => {},
+      disclose, dismiss, closeDisclosure, goBack, maximize, minimize: () => {}, requestFocus, releaseFocus,
+    });
+
+    expect(AppDelegate.isEqual(appDelegateInstance1, appDelegateInstance2)).toBe(false);
+  });
+
+  it('should return false when requestFocus does not match', () => {
+    const disclose = () => {};
+    const dismiss = () => {};
+    const closeDisclosure = () => {};
+    const goBack = () => {};
+    const maximize = () => {};
+    const minimize = () => {};
+    const requestFocus = () => {};
+    const releaseFocus = () => {};
+
+    const appDelegateInstance1 = AppDelegate.create({
+      disclose, dismiss, closeDisclosure, goBack, maximize, minimize, requestFocus, releaseFocus,
+    });
+
+    const appDelegateInstance2 = AppDelegate.create({
+      disclose, dismiss, closeDisclosure, goBack, maximize, minimize, requestFocus: () => {}, releaseFocus,
+    });
+
+    expect(AppDelegate.isEqual(appDelegateInstance1, appDelegateInstance2)).toBe(false);
+  });
+
+  it('should return false when releaseFocus does not match', () => {
+    const disclose = () => {};
```

Comment:
Total nit: could we pull these functions and `appDelegateInstance1` out such that these don't need to be rewritten for each jest test?